src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -162,7 +152,7 @@ void OnFocusedChange(bool hasFocus)
 
 			// This will eliminate additional native property setting if not required.
 			if (VirtualView.ClearButtonVisibility == ClearButtonVisibility.WhileEditing)
-				NativeView?.UpdateClearButtonVisibility(VirtualView, ClearButtonDrawable);
+				UpdateValue(nameof(VirtualView.ClearButtonVisibility));
```

Comment:
Instead of manually calling a method again, just fire of the visibility changed as this means it can now actually be overridden.